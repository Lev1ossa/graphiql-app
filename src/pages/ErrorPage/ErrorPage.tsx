export function ErrorPage(): JSX.Element {
  return (
    <>
      <header />
      <main>Error Page</main>
      <footer />
    </>
  )
}
