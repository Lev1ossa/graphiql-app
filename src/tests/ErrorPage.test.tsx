import { ErrorPage } from '@/pages/ErrorPage'
import { expect, test } from 'vitest'
import renderInContext from './renderInContext'

test('Correct ErrorPage snapshot', () => {
  const { baseElement } = renderInContext(<ErrorPage />)
  expect(baseElement).toMatchSnapshot()
})
