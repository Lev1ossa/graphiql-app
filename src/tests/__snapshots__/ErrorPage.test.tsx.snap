// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Correct ErrorPage snapshot 1`] = `
<body>
  <div>
    <header />
    <main>
      Error Page
    </main>
    <footer />
  </div>
</body>
`;
